lib/src/wallet/mnemonic.dart

```diff
@@ -16489,4 +16489,12 @@ class Mnemonic {
     final salt = Encoding.fromUtf8("mnemonic$passphrase");
     return Hash.pkcs5(password, salt, iterations: 2048);
   }
+
+  static Iterable<String> searchFor(String word,
+      [WordList language = WordList.english]) {
+    final trimmedWord = word.trim();
+    return trimmedWord.length >= 2
```

Comment:
It looks like `length >= 2` will not work for Chinese which has single character words. If single character is not performant, might need to optimize a bit, or do single character for some languages and the not others.